tests/wsl/firstrun.pm

```diff
@@ -136,8 +138,8 @@ sub run {
         # SLED Workstation license agreement
         license if (check_var('SLE_PRODUCT', 'sled'));
         # And done!
-        assert_screen ['trust_nvidia_gpg_keys', 'wsl-installation-completed'], timeout => 240;
-        send_key 'alt-t' if (match_has_tag 'trust_nvidia_gpg_keys');
+        # assert_screen ['trust_nvidia_gpg_keys', 'wsl-installation-completed'], timeout => 240;
+        # send_key 'alt-t' if (match_has_tag 'trust_nvidia_gpg_keys');
```

Comment:
Is this a leftover? Or did screen disappear again?

Reply:
This is an ongoing discussion and it seems that the screen only appears during the "development period". Also, it looks like that code was duplicated in L98 of the same file. Didn't want to remove it completely just in case it returns...

Reply:
Please add a corresponding comment then.

Reply:
Done :heavy_check_mark: